<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Shae</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="connectAction"/>
    <addaction name="resetAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="exportFootageButton"/>
    <addaction name="exportResultsButton"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSearch"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="connectAction">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="resetAction">
   <property name="text">
    <string>Reset Services</string>
   </property>
   <property name="toolTip">
    <string>Reset the services on the drone</string>
    </property>
   </action>
  <action name="exportFootageButton">
   <property name="text">
    <string>Export Footage</string>
   </property>
  </action>
  <action name="exportResultsButton">
   <property name="text">
    <string>Export Results</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
